opm/grid/cpgrid/processEclipseFormat.cpp

```diff
@@ -251,7 +251,7 @@ namespace cpgrid
         if( ccobj_.rank() != 0 )
         {
             // We do not store any grid information
-            ccobj_.scatter(logical_cartesian_size_.data(), logical_cartesian_size_.data(), 3, 0);
+            ccobj_.broadcast(logical_cartesian_size_.data(), 3, 0);
```

Comment:
this should go in the return block in the processEclipseFormat overload that calls this one.

Reply:
Turns out that this was indeed quite a mess. Thanks for the hint